examples/rsp-cra-18/src/App.test.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```

Comment:
Since this will be a sample test app for verdaccio/bundle size checks you can probably get rid of the tests + test dependencies that CRA brings in with it. If you wanna keep it for some test writing experience that is perfectly fine too